app/services/concerns/course/user_invitation_service/email_invitation_concern.rb

```diff
@@ -72,10 +72,11 @@ def invite_from_file(file)
   # @raise [CSV::MalformedCSVError] When the file provided is invalid.
   def load_from_file(file)
     invites = []
-    CSV.foreach(file, headers: true) do |row|
+
+    CSV.foreach(file, headers: ['Name', 'Email']) do |row|
       row = row.fields(0..1)
       row.unshift(row.first) unless row.length >= 2
-      invites << { name: row[0], email: row[1] } unless row[0].blank? || row[1].blank?
+      invites << { name: row[0], email: row[1] } if valid_row?(row)
```

Comment:
In that case there'll need to be a check on each loop to see if it's the first row, then treat only the first row specially.